interface/main/messages/templates/linked_documents.php

```diff
@@ -0,0 +1,237 @@
+<?php
+
+/**
+ * linked_documents.php is a sub section template for the messages.php script.  It is primarily used for displaying attached
+ * documents (usually received from the phimail-server process. It handles the preview and display of documents as well
+ * as validation of CDA types of documents if direct mail is turned on in the globals.   validation is an async process
+ * that will hit the server and validate the xsd and schematron of the CDA.  The errors are then populated and displayed
+ * on the screen with associated error counts being displayed if they are available.  Each document is handled separately
+ * as an async validation process.
+ *
+ * @package openemr
+ * @link      http://www.open-emr.org
+ * @author    Stephen Nielson <snielson@discoverandchange.com>
+ * @copyright Copyright (c) 2022 Discover and Change, Inc. <snielson@discoverandchange.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Common\Database\QueryUtils;
+use OpenEMR\Common\Twig\TwigContainer;
+use OpenEMR\Common\Logging\SystemLogger;
+use OpenEMR\Services\Cda\CdaValidateDocumentObject;
+
```

Comment:
would it make sense to do something like following to make it clear that the script will never be called directly by good/bad actors:
```
if (empty($noteid)) {
    exit;
}
```

Reply:
Should be fixed.